orc8r/gateway/c/common/service_registry/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "service_registry",
```

Comment:
I was hoping we would create only one library or at least fewer than 6 or howmany ever we have.

Reply:
@amarpad  I think it's OK to keep the libraries separate just so that we can do partial builds easier. But I do have a task tracking the work to flatten the orc8r/gateway/c/common subdirectories. (https://github.com/magma/magma/issues/8453)
I'm deferring this work to after we commit to Bazel migration. I'm trying to not affect the current CMake build at all in this iteration. :)